content/articles/2018-12-textrecipes-0-0-1.Rmarkdown

```diff
@@ -0,0 +1,84 @@
+---
+title: 'textrecipes 0.0.1'
+author: Emil Hvitfeldt
+date: '2018-12-20'
+slug: textrecipes-0-0-1
+description: >
+    textrecipes 0.0.1 is now on CRAN!
+categories:
+  - package
+photo:
+  url: https://unsplash.com/photos/X1exjxxBho4
+  author: Roman Kraft
+---
+
+```{r setup, include = FALSE}
+library(textrecipes)
+knitr::opts_chunk$set(
+  collapse = TRUE, comment = "#>", 
+  fig.width = 7, 
+  fig.align = 'center',
+  fig.asp = 0.618, # 1 / phi
+  out.width = "700px"
+)
+```
+
+We're delighted to announce the release of [textrecipes 0.0.1](https://github.com/tidymodels/textrecipes) on CRAN. [`textrecipes`](https://github.com/tidymodels/textrecipes) implements a collection of new steps for the [`recipes`](https://github.com/tidymodels/recipes) package to deal with text preprocessing. [`textrecipes`](https://github.com/tidymodels/textrecipes) is still in early development so any and all feedback is highly appreciated.
+
+You can install it by running:
+
+```{r, eval = FALSE}
+install.packages("textrecipes")
+```
+
+## New steps
+
+The steps introduces here can be split into 3 types. The steps
+
+1. converts from characters to list-columns and vice versa
```

Comment:
House style is to capitalize at the beginning of each  element of a numbered list. Since this list is part of a sentence, you should use commas after the first two items, including an Oxford comma and an "and" after the second to last bullet. The last item in the list gets a period at the end.